.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           name: Installing SmartPeak SuperBuild
           command: |
             cd ~/SmartPeak
-            #mkdir superbuild
+            mkdir -p superbuild
```

Comment:
Is this needed?  Now that we are pointing to a specific release we should not need to store ImGui in our repo correct?

Reply:
This is to make sure `superbuild` folder is newly created even if it exists so that using the latest ImGui pulled is used.

Reply:
I see.